frontend/main/src/pages/membership.tsx

```diff
@@ -293,24 +266,3 @@ export default function Membership({
     </>
   );
 }
-
-export async function getStaticProps(): Promise<{
```

Comment:
By removing this it wont run

![image](https://user-images.githubusercontent.com/3102249/111090953-cf0e0580-8507-11eb-98c1-4b6ddda5bf11.png)